client/src/components/header/header.js

```diff
@@ -161,13 +156,13 @@ const Header = (props) => {
 
                         </NavItem>
                         <NavItem
-                            class="collapse navbar-collapse"
+                            className="collapse navbar-collapse"
```

Comment:
Watch out for changing class to className. This case it is using bootstrap, and the burger menu stops working correctly when using className. Please revert the class to className changes.

Reply:
I'm sorry I didn't realize that, I'll revert the changes!